ocpmodels/common/utils.py

```diff
@@ -264,22 +264,42 @@ def _import_local_file(path: Path, *, project_root: Path) -> None:
 
 
 def setup_experimental_imports(project_root: Path) -> None:
-    experimental_folder = (project_root / "experimental").resolve()
-    if not experimental_folder.exists() or not experimental_folder.is_dir():
+    """
+    Import selected directories of modules from the "experimental" subdirectory.
+
+    By default, if present, a directory which matches $USER will automatically
+    be included. If a file named ".include" is present in the "experimental"
+    subdirectory, this will override that behavior and only those directories
+    named in the .include will be imported.
+
+    :param project_root: The root directory of the project (i.e., the "ocp" folder)
+    """
+    experimental_dir = (project_root / "experimental").resolve()
+    if not experimental_dir.exists() or not experimental_dir.is_dir():
         return
 
-    experimental_files = [
-        f.resolve().absolute() for f in experimental_folder.rglob("*.py")
-    ]
-    # Ignore certain directories within experimental
-    ignore_file = experimental_folder / ".ignore"
-    if ignore_file.exists():
-        with open(ignore_file, "r") as f:
-            for line in f.read().splitlines():
-                for ignored_file in (experimental_folder / line).rglob("*.py"):
-                    experimental_files.remove(
-                        ignored_file.resolve().absolute()
-                    )
+    experimental_files = []
+    include_file = experimental_dir / ".include"
+    curr_user = os.environ.get("USER")
+
+    if include_file.exists():
+        with open(include_file, "r") as f:
+            include_dirs = f.read().splitlines()
+
+        for inc_dir in include_dirs:
+            experimental_files.extend(
+                f.resolve().absolute()
+                for f in (experimental_dir / inc_dir).rglob("*.py")
+            )
+
+    # DSL: How do we feel about this default behavior below? Happy to remove this logic, but
+    # it felt simple and like a reasonable default. In any case, it's no more extra work than
+    # if it's not here (where everyone needs to make a .include anyway) so figured I'd throw it in...
+    elif curr_user is not None and (experimental_dir / curr_user).exists():
```

Comment:
Better to have the default behavior not import anything and explicitly define what needs to be imported in `.include` instead. Could result in silent errors if someone imports something from their directory unintentionally that conflicts with the main `ocp` registry.